src/gui/screen_printing.cpp

```diff
@@ -484,9 +462,9 @@ int screen_printing_event(screen_t *screen, window_t *window, uint8_t event, voi
             if (gui_msgbox("Are you sure to stop this printing?",
                     MSGBOX_BTN_YESNO | MSGBOX_ICO_WARNING | MSGBOX_DEF_BUTTON1)
                 == MSGBOX_RES_YES) {
-                abort_print(screen);
-                screen_close();
-                return 1;
+                marlin_print_abort();
+                //                screen_close();
+                //                return 1;
```

Comment:
Why is it commented out?

Reply:
This is because new implementation is waiting for the stop state, stop is now asynchronous (parking head...). Also i changed the behavior and after stop print you can now "reprint". Maybe it will be better than previous behavior, maybe we will return to previous behavior. We can discuse.

Reply:
I am very much against having commented code in master – we have GIT to deal with those situations. But let's leave it for now; we can discuss this again after the git training next week :)